test-app/app/templates/flatpickr-test.hbs

```diff
@@ -0,0 +1,6 @@
+truc
```

Comment:
```suggestion
```

Was this supposed to be here?

Reply:
absolutely not, it's a french word i use for testing, it means 'thing'. I'm removing this ASAP 😅